### PR TITLE
Sarah small tweaks

### PR DESCRIPTION
**Pull request by snwessel**
A couple of small tweaks based on the feedback from @liu-caro 
- Change the Workshop accent color on the schedule (from dark blue to green)
- Change the link color on the schedule (from black to light blue)
- Fix spacing on the Resources section (images had been overhanging the cards on small screens)
- Change the width of schedule event descriptions on large screens

Screenshots:
image
image
image
image
image

### CODE REVIEW

File: src/components/Schedule.js [hunk 1]
```diff
@@ -182,7 +182,9 @@ const Schedule = () => {
                             </Accordion.Toggle>
                             <Accordion.Collapse eventKey={idx.toString()}>
                               <Card.Body>
-                                <p>{event.description}</p>
+                                <Col lg="8">
```
Comment (snwessel) [inline src/components/Schedule.js:200]: The description will now take up the full space on medium or smaller screens, and 8/12ths of the space on large screens

File: src/styles/schedule.scss [hunk 2]
```diff
@@ -157,8 +157,13 @@
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
-      color: #000;
       margin-bottom: 0;
+      color: $teal-blue;
+    }
+    &__zoom:hover {
+      color: $teal-blue !important;
+      font-weight: bold;
```
Comment (snwessel) [inline src/styles/schedule.scss:165]: Links bold on hover - we do that in the footer and I thought it looked nice

**Review by ghost — COMMENTED** [review]
Screen Shot 2021-02-16 at 3 01 20 PM

The resources cards look a little off center on medium/small screens – I think there's a funky combination of padding and margin on resources__card-label? also the "K" of scrapbook seems to go onto the next line for some reason (doesn't happen on prod)

**Comment by snwessel** [comment]
@bugsalexander oop you're right thanks for catching that! I had fixed that when I was taking screenshots and then forgot to push my change 🤦

**Review by ghost — APPROVED** [review]
Yay, looks great!!

**Review by anikarabenhorst — APPROVED** [review]
lgtm :)

**Pull request merged**